src/geometry/interior.cpp

```diff
@@ -85,15 +85,19 @@ auto point_to_polygon_dist(const point<T>& point, const polygon<T>& polygon)
 
     for (const auto& ring : polygon)
     {
-        for (std::size_t i = 0, len = ring.size(), j = len - 1; i < len; j = i++)
+        std::size_t length = ring.size();
+        if (length > 1)
```

Comment:
without this check, `len - 1` overflows when `len == 0`

Reply:
I don't think this is a problem. That overflown `len - 1` value is not used when the ring is empty.

Reply:
@talaj - yes, true, but sanitizer wasn't happy so I added the check. I'm off on vacation right now, feel free to revert or work-around differently.

Reply:
I think it's fine, thanks for the info.